### DISCUSSION TITLE
list resources in a Resource ACL via CLAPI

### DISCUSSION THREAD

**Discussion by matletix** [centreon-archive/centreon-archived#9954] [Q&A]
Hi, 

Is it possible to list the Hosts/Hostsgroups/Excluded hosts that are associated with a Resource ACL via the CLAPI or any other API ?
I saw there is the possibility to add/remove them via the CLAPI with the grant/revoke action (https://docs.centreon.com/current/en/api/clapi.html#grant-and-revoke-2) but I didn't find a way to list the already present objects.

**Comment by lpinsivy** [comment]
CLAPI does not actually allow you to list the resources

**Reply by matletix** [reply]
@lpinsivy Ok, is there a reason to this ? Or is it just not implemented ?

**Reply by lpinsivy** [reply]
not implemented.
We will keep this in mind for next APIv2 about ACL
